### ISSUE TITLE
is it possible to change the default logo on aac in a specific realm / client pages?

### ISSUE THREAD

**Issue by gianpaolo-tndigit** [scc-digitalhub/AAC#73]
I looked in the customizations screen but I didn't find the possibility to "load" an image.
In the templates (attribute / errors / registrations ) folders I found a "fixed" reference:
`<img class="" th:src="@{/img/ds_logo.png}" th:alt="${displayName}">`

from documentation I found:

> Given the global frame, certain parts of the user interface can be **customized** per *realm* or per *client*, for example the header/footer combo or the application name and logo. When properly done, such customizations will increase the trustiness of the system, bridging the gap between user facing applications and the IdP interface.

**Comment by matteo-s** [comment]
Yes, the logo is customizable but the feature is not merged yet.
I'll leave this open until the feature lands in 4.x

**Comment by gianpaolo-tndigit** [comment]
Realm-level custom logos are available since commit a9c1da37 (upload via dev console, served at `/-/{realm}/logo`). 
Per-client logos can be tracked separately if still needed.
